src/libs/NetworkConnection.ts

```diff
@@ -13,7 +13,7 @@ let hasPendingNetworkCheck = false;
 
 // Holds all of the callbacks that need to be triggered when the network reconnects
 let callbackID = 0;
-const reconnectionCallbacks: Record<string, () => Promise<void>> = {};
+const reconnectionCallbacks: Record<string, () => void> = {};
```

Comment:
Why isn't this a `promise` anymore?

Reply:
Because the callback passed to `reconnect` function is not a `Promise`
https://github.com/Expensify/App/blob/d6c824b90dd252d0e4511ab07380cff986c49745/src/libs/Navigation/AppNavigator/AuthScreens.tsx#L156-L162